### PR TITLE
FRW-1101: Enabled CI checks

### PR DESCRIPTION
**Pull request by olhalivitchuk**
- Developer(s): @olhalivitchuk

- Ticket: https://spryker.atlassian.net/browse/FRW-1101

- merge: merge

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   AuthorizationPickingAppBackendApi  | patch {skip} (currently 0.x)  |                       |

### CODE REVIEW

**Comment by olhalivitchuk** [comment]
@dereuromark I was not managed to run `dev:ide-auto-completion:generate` because it failed with an unknown `APPLICATION_STORE` despite the fact that it's in `ci.yml`. I tried a few different ways to pass it but none succeeded. 

Is it possible to skip this part for the current eco module?

**Comment by dereuromark** [comment]
Maybe the helper needs a fix to get that constant working
But sure, can be skipped for now

**Review by yaroslav-spryker — APPROVED** [review]

**Review by dimitriyTsemma — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -81,8 +81,47 @@ jobs:
...
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: '8.0'
+                  extensions: mbstring, intl, bcmath
+
+            - name: Composer prefer lowest
+              run: composer lowest-setup
+
+            - name: Run PHPStan (currently not running properly)
               run: composer stan || true
```
Comment (asmarovydlo) [inline .github/workflows/ci.yml:124]: Can we avoid `|| true`?

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (olhalivitchuk) [reply .github/workflows/ci.yml:124]: It's for all eco modules, there is an error that does not depend on us. I do not remember the exception but can remove it and show the result if it's needed

Comment (dereuromark) [reply .github/workflows/ci.yml:124]: Those hacks make CI green without it being green, I would also suggest those are only valid temporary to see if follow up tasks are OK
those should not get merged to master.

Comment (dereuromark) [reply .github/workflows/ci.yml:124]: > PHP Fatal error:  Uncaught TypeError: _HumbugBox5f943a942674\Nette\Utils\Reflection::getUseStatements(): Return value must be of type array, null returned in phar:///home/runner/work/authorization-picking-app-backend-api/authorization-picking-app-backend-api/vendor/phpstan/phpstan/phpstan/vendor/nette/utils/src/Utils/Reflection.php:180

Maybe PHPStan needs to be removed from prefer lowest if it isnt running

Comment (olhalivitchuk) [reply .github/workflows/ci.yml]: Do you know which part in ci.yml is responsible for generating Transfers ets ?

Comment (dereuromark) [reply .github/workflows/ci.yml]: tests do that automatically via helpers afaik

thats why some modules have tests || true for phpstan to run afterwards

Comment (dereuromark) [reply .github/workflows/ci.yml]: This is documented in the standalone testing doc for
\SprykerTest\Shared\Transfer\Helper\TransferGenerateHelper

Comment (dereuromark) [reply .github/workflows/ci.yml]: That said: The whole testing harness for Spryker modules is a beast and could definitly need some serious refactor/rework

Comment (olhalivitchuk) [reply .github/workflows/ci.yml]: With `tests || true` and some dependency adjustments works, thanks!
@dereuromark and @asmarovydlo please check again

**Review by dereuromark — APPROVED** [review]

**Pull request merged**